src/components/organization/OrganizationsPartnerList.js

```diff
@@ -0,0 +1,154 @@
+import {Controller, useForm} from "react-hook-form";
+import {Box, Button, FormControl, Grid, IconButton, InputLabel, MenuItem, Select, TextField} from "@material-ui/core";
+import {withSwal} from "../formsCommons/WithSwal";
+import React, {useEffect} from "react";
+import ClearIcon from "@material-ui/icons/Clear";
+import {OrganizationPartner} from "./OrganizationPartner";
+import {usersAPI} from "../../utils/apis/UserApi";
+
+export const OrganizationsPartnerList = (props) => {
+
+    const id = 1 //TODO change after jwt
```

Comment:
Może wyjdzie, że się czepiam, ale po co to? Jeśli robiłeś na wzór ofert, to niepotrzebnie, bo oferty mają dwa use casy, do jednego jest potrzebne id, ale do drugiego nie bardzo. A jeśli tego id potrzebujesz tylko do use Effecta, to równie dobrze będziesz mógł użyć jwt tam, a nie tworzyć nową zmienną i patrzeć, czy jest defined, czy nie.

Reply:
```typescript
//TODO change after jwt
```